crates/storage/db/src/tables/mod.rs

```diff
@@ -160,30 +160,30 @@ macro_rules! tables {
 
 tables!([
     (CanonicalHeaders, TableType::Table),
-    (HeaderTD, TableType::Table),
+    (HeaderTerminalDifficulties, TableType::Table),
     (HeaderNumbers, TableType::Table),
     (Headers, TableType::Table),
     (BlockBodyIndices, TableType::Table),
     (BlockOmmers, TableType::Table),
     (BlockWithdrawals, TableType::Table),
-    (TransactionBlock, TableType::Table),
+    (TransactionBlocks, TableType::Table),
     (Transactions, TableType::Table),
-    (TxHashNumber, TableType::Table),
+    (TransactionHashNumbers, TableType::Table),
     (Receipts, TableType::Table),
     (PlainAccountState, TableType::Table),
     (PlainStorageState, TableType::DupSort),
     (Bytecodes, TableType::Table),
-    (AccountHistory, TableType::Table),
-    (StorageHistory, TableType::Table),
-    (AccountChangeSet, TableType::DupSort),
-    (StorageChangeSet, TableType::DupSort),
-    (HashedAccount, TableType::Table),
-    (HashedStorage, TableType::DupSort),
+    (AccountHistories, TableType::Table),
+    (StorageHistories, TableType::Table),
```

Comment:
same as for stage names, this should be singular because we're indexing the history of account and storage changes, and not multiple histories. But maybe making `Account` and `Storage` plural is a good idea
```suggestion
    (AccountsHistory, TableType::Table),
    (StoragesHistory, TableType::Table),
```